buzzmobile/plan/steering/steering.py

```diff
@@ -41,7 +41,7 @@
 BRAKING_DISTANCE = rospy.get_param('braking_distance')
 THRESHHOLD = rospy.get_param('braking_score_threshhold')
 BUZZMOBILE_WIDTH = rospy.get_param('buzzmobile_width')
-MAX_SPEED = rospy.get_param('max_speed', 2.0) #TODO don't be a murder machine
+MAX_SPEED = rospy.get_param('max_speed', 1.0) #TODO measure car max speed
```

Comment:
Yes. And I think that if rosparam server isn't up, this *should* throw an exception and error out.